database/dtb.ksy

```diff
@@ -0,0 +1,127 @@
+meta:
+  id: dtb
+  title: Flattened Devicetree Format
+  file-extension: dtb
+  application:
+    - Linux
+    - Das U-Boot
+  xref:
+    wikidata: Q16960371
+  license: CC0-1.0
```

Comment:
It is probably a good idea to add `ks-version: 0.9`, because the `doc-ref` key allows a list as of 0.9:

https://github.com/kaitai-io/kaitai_struct_formats/blob/c4551e5a3d5cafdbee23c885c78a795d42380397/database/dtb.ksy#L31-L34

The `/meta/ks-version` is supposed to check if the compiler version is sufficient, so that the user gets a clear error message

```
/meta: this ksy requires compiler version at least 0.9, but you have 0.8
```

and not cryptic

```
/doc-ref: expected string, got List(https://github.com/devicetree-org/devicetree-specification/releases/
tag/v0.3, https://github.com/devicetree-org/devicetree-specification/blob/ba2aa67/source/flattened
-format.rst, https://elinux.org/images/f/f4/Elc2013_Fernandes.pdf) (class scala.collection.immutable
.$colon$colon)
```

which looks rather like a bug.